sagemaker-clarify/computer_vision/object_detection/object_detection_clarify.ipynb

```diff
@@ -296,6 +297,31 @@
     "net.hybridize()  # hybridize to optimize computation"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get images from the Caltech 256 dataset"
```

Comment:
nit: Please rephrase this line as "Get RGB images from Caltech 256 dataset"

Reply:
Agreed. I think it's worth adding a note (either here or at the inference section) saying something like: "Note that this model only supports inference on color images, not black-and-white/grayscale."